### PR TITLE
update tck README

### PR DESCRIPTION
**Pull request by shenghaiyang**

### CODE REVIEW

**Comment by viktorklang** [comment]
Thank you!

**Comment by viktorklang** [comment]
Hi @shenghaiyang,

Would you mind rebasing on top of current HEAD?

**Comment by viktorklang** [comment]
(I'm seeing some weird Travis/Gradle issues: https://travis-ci.org/reactive-streams/reactive-streams-jvm/jobs/274536357)

**Comment by akarnokd** [comment]
JDK 6 is no longer supported. The HEAD should contain the correct CI settings.

**Comment by viktorklang** [comment]
Hi @shenghaiyang, your PR unfortunately needs to be reset.
Could you reset your master branch to origin/master and then cherry-pick commit 37324b6af4492a56d0c5c7ed92ae35898d4ba394 to it and update the PR?

**Comment by viktorklang** [comment]
@shenghaiyang Alternatively, you set your master branch to be https://github.com/reactive-streams/reactive-streams-jvm/tree/pr/392 and push that to your repo (which would update the pr)

**Comment by shenghaiyang** [comment]
@viktorklang Thank you very much!

File: tck/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Since the TCK is aimed at Reactive Stream implementers, looking into the sources
 In order to make mapping between test cases and Specification rules easier, each test case covering a specific
 Specification rule abides the following naming convention: `TYPE_spec###_DESC` where:
 
-* `TYPE` is one of: [#type-required](required), [#type-optional](optional), [#type-stochastic](stochastic) or [#type-untested](untested) which describe if this test is covering a Rule that MUST or SHOULD be implemented. The specific words are explained in detail below.
+* `TYPE` is one of: [required](#type-required), [optional](#type-optional), [stochastic](#type-stochastic) or [untested](#type-untested) which describe if this test is covering a Rule that MUST or SHOULD be implemented. The specific words are explained in detail below.
```
Comment (ktoso) [inline tck/README.md]: LGTM, could you apply the same fix in https://github.com/reactive-streams/reactive-streams-jvm/blob/master/tck-flow/README.md ?

Comment (shenghaiyang) [reply tck/README.md]: ok :-)

**Comment by ktoso** [comment]
Thanks, LGTM :)

**Comment by viktorklang** [comment]
Awesome, thank you @shenghaiyang!

**Pull request merged**